azure-pipelines.yml

```diff
@@ -40,7 +40,14 @@ steps:
 - script: dotnet build --configuration $(buildConfiguration)
   displayName: 'dotnet build $(buildConfiguration)'
 
-- script: dotnet-sonarscanner end /d:sonar.login="$(sonar.token)"
+- task: DotNetCoreCLI@2
+  displayName: 'dotnet test $(buildConfiguration)'
+  inputs:
+    command: test
+    projects: '**/*.sln'
+    arguments: '--configuration $(buildConfiguration) --collect "Code coverage"'
+
+- script: dotnet-sonarscanner end /d:sonar.login="$(sonar.token) /d:sonar.verbose=true"
```

Comment:
There are two problems with this line:
* the quoting is wrong
* specifying sonar.verbose isn't valid in the `end` step.

Just remove the `/d:sonar.verbose=true` and it will be correct.

This would cause the build to fail if it was getting this far (but it isn't).